### PR TITLE
add support for signing mlbf via autograph

### LINKED ISSUE #13617: sign MLBF kinto attachments
for MLBF attachments (#13616) to be trusted we need to sign them and include the signature in the kinto record submission.

### PR DESCRIPTION
**Pull request by eviljeff**
fixes #13617 ... assuming my implementation is correct.  

(Also contains a minimal implementation for #13695 that should be close enough for now)

### CODE REVIEW

File: scripts/autograph_travis_test_config.yaml [hunk 1]
```diff
@@ -240,11 +240,25 @@ signers:
           PIDNiTxNecePOmrD+1ivAEXcoL+e1w==
           -----END PRIVATE KEY-----
 
+    - id: addon-blocklist_content-signature_mozilla_org
```
Comment (eviljeff) [inline scripts/autograph_travis_test_config.yaml]: copied from https://github.com/mozilla-services/autograph/blob/master/autograph.yaml#L43 tbh

File: src/olympia/blocklist/cron.py [hunk 2]
```diff
@@ -20,12 +22,22 @@ def upload_mlbf_to_kinto():
     server = KintoServer(KINTO_BUCKET, KINTO_COLLECTION_MLBF)
     stats = {}
     key_format = get_mlbf_key_format()
+    signing_datetime = int(time.time() * 1000)
```
Comment (EnTeQuAk) [inline src/olympia/blocklist/cron.py]: You need to explain to an old man what this actually doing, I'm not sure I've seen time.time() being multiplied by `1000` before :thinking:

Comment (eviljeff) [reply src/olympia/blocklist/cron.py]: heh, it's just to get time in milliseconds, but as a integer.  Kinto uses the same timestamp format for the record `modified` property.

**Comment by eviljeff** [comment]
On the back of a recent meeting the current thinking is that signing the bloomfilter is ineffective against the kinds of attacks we'd want to prevent so closing this for now 😞

**Pull request closed without merging**